commit 60af6b1a3c304e8704d7f21a354bad89b629d897 (refs/changes/31/1431/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-02-03T15:33:38-08:00 (2 years, 8 months ago)
    
    IMGAPI-613 the 'import_image' task of the import-remote-image workflow has too short of a timeout
    Reviewed by: Josh Wilsdon <josh@wilsdon.ca>
    Reviewed by: Orlando Vazquez <orlando@joyent.com>

diff --git a/lib/workflows/import-remote-image.js b/lib/workflows/import-remote-image.js
index 150a6aa..10142f5 100644
--- a/lib/workflows/import-remote-image.js
+++ b/lib/workflows/import-remote-image.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2014, Joyent, Inc.
+ * Copyright 2017 Joyent, Inc.
  */
 
 /*
@@ -18,7 +18,7 @@ var sdcClients = require('sdc-clients');
 var async = require('async');
 var imgapiUrl;
 
-var VERSION = '7.0.6';
+var VERSION = '7.0.7';
 
 
 
@@ -196,7 +196,7 @@ var workflow = module.exports = {
         }
     }, {
         name: 'import_image',
-        timeout: 10,
+        timeout: 60,
         retry: 1,
         body: importImage,
         modules: { sdcClients: 'sdc-clients' }
@@ -216,13 +216,13 @@ var workflow = module.exports = {
     timeout: 7250,
     onerror: [ {
         name: 'delete_image',
-        timeout: 10,
+        timeout: 60,
         retry: 1,
         body: deleteImage,
         modules: { sdcClients: 'sdc-clients' }
     }, {
         name: 'origin_delete_images',
-        timeout: 10,
+        timeout: 60,
         retry: 1,
         body: originDeleteImages,
         modules: {
